devicetypes/smartthings/aeon-home-energy-meter.src/aeon-home-energy-meter.groovy

```diff
@@ -92,29 +133,54 @@ def zwaveEvent(physicalgraph.zwave.Command cmd) {
 }
 
 def refresh() {
+	log.debug "refresh()..."
 	delayBetween([
-		zwave.meterV2.meterGet(scale: 0).format(),
-		zwave.meterV2.meterGet(scale: 2).format()
+		encap(zwave.meterV2.meterGet(scale: 0)),
+		encap(zwave.meterV2.meterGet(scale: 2))
 	])
 }
 
 def reset() {
+	log.debug "reset()..."
 	// No V1 available
-	return [
-		zwave.meterV2.meterReset().format(),
-		zwave.meterV2.meterGet(scale: 0).format()
-	]
+	delayBetween([
+		encap(zwave.meterV2.meterReset()),
+		encap(zwave.meterV2.meterGet(scale: 0))
+	])
 }
 
 def configure() {
-	def cmd = delayBetween([
-		zwave.configurationV1.configurationSet(parameterNumber: 101, size: 4, scaledConfigurationValue: 4).format(),   // combined power in watts
-		zwave.configurationV1.configurationSet(parameterNumber: 111, size: 4, scaledConfigurationValue: 300).format(), // every 5 min
-		zwave.configurationV1.configurationSet(parameterNumber: 102, size: 4, scaledConfigurationValue: 8).format(),   // combined energy in kWh
-		zwave.configurationV1.configurationSet(parameterNumber: 112, size: 4, scaledConfigurationValue: 300).format(), // every 5 min
-		zwave.configurationV1.configurationSet(parameterNumber: 103, size: 4, scaledConfigurationValue: 0).format(),    // no third report
-		zwave.configurationV1.configurationSet(parameterNumber: 113, size: 4, scaledConfigurationValue: 300).format() // every 5 min
-	])
-	log.debug cmd
-	cmd
+	log.debug "configure()..."
+	if (zwaveInfo.model.equals("005F"))
+		delayBetween([
+			// Reset the device to the default settings
+			encap(zwave.configurationV1.configurationSet(parameterNumber: 255, size: 4, scaledConfigurationValue: 1))
```

Comment:
Looking at the manual it's not entirely clear what the default reporting interval is for the first two groups. Is it 5 minutes?

Reply:
Group 1 defaults to 1 minute, Group 2 is 2 minutes.

Reply:
Resetting device to default configuration seems to not work, or "default" is different from what is stated in the manual. I'm getting reports every 5 second, so I think that all of important configurations we should do anyway and don't look at the 'defaults'.